test/pg-cdc-resumption/configure-postgres.td

```diff
@@ -8,8 +8,8 @@
 # by the Apache License, Version 2.0.
 
 $ postgres-execute connection=postgres://postgres:postgres@postgres
-CREATE USER debezium WITH SUPERUSER PASSWORD 'debezium';
```

Comment:
The debezium user was previously unused, it was some copypasta from another test I guess.